app/routes/chatbot.js

```diff
@@ -36,15 +35,15 @@ router.use(requiredParamsMiddleware(requiredParamsConf));
 router.use(userIncomingMessageMiddleware(userIncomingMessageConf));
 router.use(mapRequestParamsMiddleware(mapParamsConf));
 
-/**
- * Check if DS User exists for given mobile number.
- */
-router.use(lookUpUserMiddleware());
-
-/**
- * Create DS User for given mobile number if we didn't find one.
- */
-router.use(createNewUserIfNotFoundMiddleware());
+router.use(
+  /**
+   * Check if DS User exists for given mobile number.
+   */
+  getUserMiddleware(),
+  /**
+   * Create DS User for given mobile number if we didn't find one.
+   */
+  createNewUserIfNotFoundMiddleware());
```

Comment:
I'm testing creating a new User locally, and can't figure out why I'm seeing this Retry message in the logs. Do you get the same @rapala61 ?
```
12:11:08 PM web.1 |  info: Receiving: incoming keyword
12:11:08 PM web.1 |  debug: contentful.fetchKeyword:luckybot
12:11:08 PM web.1 |  debug: contentful.fetchSingleEntry:{"content_type":"keyword","fields.keyword":"luckybot"}
12:11:08 PM web.1 |  info: /v1/chatbot post:{"incoming_image_url":"http://www.fox.com/sites/default/files/X-Files_hero.png","keyword":"luckybot"}
12:11:08 PM web.1 |  debug: User.lookup(mobile, 201706191211)
12:11:09 PM web.1 |  info: User.lookup could not find mobile:201706191211
12:11:09 PM web.1 |  debug: User.post
12:11:10 PM web.1 |  info: northstar.Users created user:594821cd10707d08a62063f3
12:11:10 PM web.1 |  info: This is a retry request. Retry count: 1
12:11:10 PM web.1 |  debug: phoenix.fetchCampaign:2900
```
I can replicate it by using a different new mobile number each time.